examples/pipelines/ccfraud/config.yaml

```diff
@@ -0,0 +1,53 @@
+# example yaml config
+
+# using this to store references to Azure ML
+aml:
+  subscription_id: "<SUBSCRIPTION_ID>"
+  resource_group_name: "<RESOURCE_GROUP>"
+  workspace_name: "<AML_WORKSPACE_NAME>"
+
+# federated learning parameters
+federated_learning:
+  orchestrator:
+    compute: cpu-orchestrator
+    datastore: datastore_orchestrator
+
+  silos:
```

Comment:
Nevermind, we talked and now I do understand that the silos are actually using different data. It is okay to leave as-is for this PR, but eventually we want to decouple the splitting job from the training job, since the graph just shows 2 datasets and that will be confusing to the users.